.github/workflows/sync_peloton_to_garmin.yml

```diff
@@ -0,0 +1,80 @@
+# This is a workflow that will automatically sync your Peloton rides with Garmin
+
+name: Sync workflow
+
+# Controls when the action will run.
+on:
+  workflow_dispatch:
+    inputs:
+      workoutsToDownload:
+        type: number
+        default: "5"
+  schedule:
+    - cron: 0 1 * * *
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    # The type of runner that the job will run on
+    container: 
+      image:  dkuwahara/peloton-to-garmin:latest

```

Comment:
1. Should this be `philosowaffle/peloton-to-garmin` instead?

2. Recommend setting this to the `stable` tag. I have a habit of breaking things on `latest`, so prefer it to be something a user explicitly chooses to use vs. the default.